model/metricset/event_test.go

```diff
@@ -130,6 +131,19 @@ func TestDecode(t *testing.T) {
 		}
 	}
 }
+func TestDecodeV1(t *testing.T) {
```

Comment:
this is parameterizes `testDecode` while in "span" the decode tests are simply copied. I'm fine either way, but we should consider aligning them.

Reply:
So you're OK with the current changes?

Reply:
yes fine with me